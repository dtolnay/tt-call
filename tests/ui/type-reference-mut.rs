#[macro_use]
extern crate tt_call;

tt_call! {
    macro = [{ parse_type }]
    input = [{ &mut }]
}

fn main() {}
