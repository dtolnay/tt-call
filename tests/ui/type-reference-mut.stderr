error: no rules expected the token `mut`
 --> tests/ui/type-reference-mut.rs:5:17
  |
5 |     input = [{ &mut }]
  |                 ^^^ no rules expected this token in macro call
  |
  = note: while trying to match end of macro
